README.md

```diff
@@ -1,59 +1,265 @@
+[here](https://symfony.com/blog/new-in-symfony-4-2-define-env-vars-per-environment)
+
+### Verify configuration
+
+To verify that your configuration works, after adding your client credentials on the `.env` file
+run the following on the command line:
+```sh
+bin/console commercetools:project-info
+```
+If everything is set up correctly, this should return
+the details of your project. For example:
+
+```
+Project's key: super-cool-project-4
+Project's name: my eshop at commercetools
+Countries: DE, US
+Currencies: EUR
+Languages: en
+Created at: 2019-02-04T11:28:49+00:00
+Messages: disabled
+```
+
+
+## Usage
+
+The commercetools Symfony bundle consists of 8 smaller bundles (The dependencies mentioned are
+only the additional ones, required for bundle-specific functionalities)
+
+- CartBundle
+- CatalogBundle
+- CtpBundle
+- CustomerBundle
+    - Dependency: `symfony/security-bundle`
+- ReviewBundle
+- SetupBundle
+    - Dependency: `symfony/console`
+- ShoppingListBundle
+- StateBundle
+    - Dependency: `symfony/console`, `symfony/workflow`, `twig/extensions`
+    
+
+By default, `CtpBundle` & `CustomerBundle` are enabled for all environments.
+`SetupBundle` & `StateBundle` are only enabled for for development environments. To
+see which bundles are enabled and optionally enable or disable them, edit
+the `config/bundles.php` file.
+
+
+### Available services
+
+The main idea is that in each Bundle there are some reusable services that you
+may inject directly in your app. As a generic pattern there are a 
+couple of `*Manager` services, that provide related actions. So, for example in `CartManager`
+you will find helpers like `getCart`, `createCartForUser` and `update`. The `update` service
+returns a  `CartUpdateBuilder` where you can dynamically build update actions. To autowire
+a service, for example `CatalogManager`, in a controller action, do the following: 
 
 ```php
-class AppKernel extends Kernel
+<?php
+namespace App\Controller;
+
+use Commercetools\Symfony\CatalogBundle\Manager\CatalogManager;
+use Symfony\Component\HttpFoundation\Response;
+
+class MyController
 {
-    public function registerBundles()
+    public function index(CatalogManager $manager)
     {
-        $bundles = [
-            // ...
-            new Commercetools\Symfony\CtpBundle\CtpBundle(),
-            // ...
-        ];
-        // ...
+        /** @var CategoryCollection $categories */
+        $categories = $manager->getCategories('en');
+
+        return new Response(
+            '<html><body>categories: '
+            // handle CategoryCollection $categories
+            .'</body></html>'
+        );
     }
-    // ...
+}
 ```
 
-Open App/config/parameters.yml.dist and add these lines
-```yaml	
-    commercetools.client_id: ~
-    commercetools.client_secret: ~
-    commercetools.project: ~
-```
+#### Available services
+
+- CartBundle: Provides helpers for carts, orders, payments and shipping-methods
+    - CartManager
```

Comment:
Instead of every service, I added a small description in each bundle, because I'm afraid it would be too repetitive. Like "provides helpers for something", "provides helpers for something else" etc. wdyt?